app/Connect/Actions/InjectPatchClientSupportLevelDataAction.php

```diff
@@ -60,9 +60,15 @@ private function buildClientSupportWarningAchievement(ClientSupportLevel $client
         return [
             'ID' => Achievement::CLIENT_WARNING_ID,
             'MemAddr' => '1=1.300.', // pop after 5 seconds
-            'Title' => ($clientSupportLevel === ClientSupportLevel::Outdated) ?
-                'Warning: Outdated Emulator (please update)' : 'Warning: Unknown Emulator',
-            'Description' => 'Hardcore unlocks cannot be earned using this emulator.',
+            'Title' => match ($clientSupportLevel) {
+                ClientSupportLevel::Outdated => 'Warning: Outdated Emulator (please update)',
+                ClientSupportLevel::Unsupported => 'Warning: Unsupported Emulator',
+                default => 'Warning: Unknown Emulator',
+            },
+            //            'Description' => 'Hardcore unlocks cannot be earned using this emulator.',
+            'Description' => ($clientSupportLevel === ClientSupportLevel::Outdated) ?
+                'Starting in April, all unlocks from this emulator version will be softcore' :
+                'Starting in April, all unlocks from this emulator will be softcore',
```

Comment:
Do you think it would make sense to check if the current date is greater than or equal to May 1 2025, and if so, change this message?

Reply:
There's going to be another PR to actually enable the downgrading (see TODOs in dorequest.php). We should just change the message then.